priya/javascript/Assignment7/JsAssign7Q3.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Bubble_sort</title>
+    </head>
+    <body>
+        <p style="text-align: center;">
+            <h4>Bubble_sort</h4>
+            Number_of_student : <input type="text" id="num" placeholder="Enter number of students"><br>
+            <br>
+            <button onclick="bubble()">submit</button><br>
+            <br>
+            <p id="demo"></p>
+        </p>
+        <script>
+            function bubble(){
+                var input,arr,temp;
+                input=parseInt(document.getElementById("num").value);
+                arr=[];
+                for (var i=0; i<input ; i++){
+                    arr[i]=parseInt(prompt("Enter the marks"))
+                }
+                document.write("********* Before Sorting (Actual Marks) ********** <br>" );
+                for(var i=0; i< input ; i++){
+                    document.write("Marks "+ (i+1)+ ": "+arr[i] + "<br>")
+                }
+                document.write( "<br>" )
+                document.write("********* Before Sorting ********** <br>" );
+                document.write( "<br>" + arr +"<br>")
+                document.write("<br>***************************************" );
+                for (var i=0; i< arr.length-1; i++){
+                    for(var j=0; j<= arr.length-1; j++){
+                        if (arr[j] > arr[j+1]){
+                            temp = arr[j];
+                            arr[j]=arr[j+1];
+                            arr[j+1]=temp;
+                        }
+                    }
+                    document.write( "<br>" )
```

Comment:
no need to write extra lines for break tag you can add it in single document.write

Reply:
changes done